templates/format-drives.j2

```diff
@@ -44,8 +44,6 @@ else
 
   [[ -e ${MDADM_CONF} ]] && cp ${MDADM_CONF} ${MDADM_CONF}.backup
   install -Dv /dev/null ${MDADM_CONF}
-  echo DEVICE ${drives} | tee ${MDADM_CONF}
```

Comment:
How/where do we setup MDADM_CONF if we are no longer writing the file here?

Reply:
2 lines later 😉 
`mdadm --examine --scan | tee -a ${MDADM_CONF}`

Reply:
Ah, right!